CHANGELOG/CHANGELOG-0.5.md

```diff
@@ -0,0 +1,28 @@
+# Changelog 0.5
+
+## 0.5.0 - 2023-03-17
+
+### Added
+
+- Added Theia Manager resync at startup. ([#150](https://github.com/antrea-io/theia/pull/150), [@yanjunz97])
+  * Support removing stale Spark jobs and database entries after restarting Theia Manager.
+  * Added backed-up Network Policy Recommendations jobs to the sync list.
+- Service Dependency Graph. ([#142](https://github.com/antrea-io/theia/pull/142), [@Dhruv-J])
```

Comment:
Added?

Reply:
You forgot to update this one?

Reply:
"Scheduled Network Policy Recommendations jobs are added back to the periodical sync list"

Reply:
I meant should we say: "Added Service Dependency Graph" to be consistent with other items.

Reply:
done